internal/acctest/oauthtokenexchangegeneratorsettings/oauth_token_exchange_generator_settings_test.go

```diff
@@ -0,0 +1,90 @@
+package acctest_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/acctest"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/provider"
+)
+
+// Attributes to test with. Add optional properties to test here if desired.
+type oauthTokenExchangeGeneratorSettingsResourceModel struct {
+	defaultGeneratorGroupRefId string
+}
+
+func TestAccOauthTokenExchangeGeneratorSettings(t *testing.T) {
+	resourceName := "myOauthTokenExchangeGeneratorSettings"
+	initialResourceModel := oauthTokenExchangeGeneratorSettingsResourceModel{
+		defaultGeneratorGroupRefId: "exampleGeneratorGroup",
+	}
+
+	updatedResourceModel := oauthTokenExchangeGeneratorSettingsResourceModel{
+		defaultGeneratorGroupRefId: "exampleGeneratorGroup2",
+	}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() { acctest.ConfigurationPreCheck(t) },
+		ProtoV6ProviderFactories: map[string]func() (tfprotov6.ProviderServer, error){
+			"pingfederate": providerserver.NewProtocol6WithError(provider.NewTestProvider()),
+		},
+		Steps: []resource.TestStep{
+			{
+				Config: testAccOauthTokenExchangeGeneratorSettings(resourceName, initialResourceModel),
+				Check:  testAccCheckExpectedOauthTokenExchangeGeneratorSettingsAttributes(initialResourceModel),
+			},
+			{
+				Config: testAccOauthTokenExchangeGeneratorSettings(resourceName, updatedResourceModel),
+				Check:  testAccCheckExpectedOauthTokenExchangeGeneratorSettingsAttributes(updatedResourceModel),
+			},
+			{
+				// Test importing the resource
+				Config:            testAccOauthTokenExchangeGeneratorSettings(resourceName, initialResourceModel),
```

Comment:
I don't think it really matters for this test, but for our usual test steps this should use updatedResourceModel

Reply:
updated